src/main/java/gui/panes/ManualConfigPane.java

```diff
@@ -0,0 +1,407 @@
+     * @param vControl the videoController
+     * @param rControl the roomController
+     */
+    public ManualConfigPane(final VideoController vControl, final RoomController rControl) {
+        this.videoController = vControl;
+        this.roomController = rControl;
+        this.manualStage = new Stage();
+        this.fillInPane = new GridPane();
+
+        final int fontSize = 20;
+        this.font = new Font(fontSize);
+    }
+
+    /**
+     * Creates the manualConfig stage and panes.
+     * @param manual the menuItem
+     */
+    public void createManualConfig(final MenuItem manual, final Stage primaryStage) {
+        manual.setOnAction(t -> {
+            if (videoController.isClosed()) {
+                manualStage.setTitle("Manual Escape Room Configuration");
+//                manualStage.initModality(Modality.APPLICATION_MODAL);
+//                manualStage.initOwner(primaryStage);
+
+                fillInPane.setAlignment(Pos.CENTER);
+
+                createPaneItems();
+                ScrollPane scrollPane = createScrollPane();
+
+                final int width = 500;
+                final int height = 500;
+                Scene manualConfigScene = new Scene(scrollPane, width, height);
+
+                manualStage.setScene(manualConfigScene);
+                manualStage.show();
+            }
+        });
+    }
+
+    /**
+     * Creates the items for manual config pane.
+     */
+    private void createPaneItems() {
+        final TextField playerField = new TextField();
+        final TextField chestField = new TextField();
+        final TextField totalDurationField = new TextField();
+
+        createPlayersLine(playerField);
+        createChestsLine(chestField);
+        createTotalDurationLine(totalDurationField);
+
+        Button proceed = new Button("Proceed");
+        proceed.setFont(font);
+
+        proceedSetUp(playerField, chestField, totalDurationField, proceed);
+
+        final int rowIndex = 3;
+        fillInPane.add(proceed, 0, rowIndex);
+    }
+
+    /**
+     * Adds the line to the pane to fill in the amount of players.
+     * @param playerField the field in which the amount of players need to be filled in
+     */
+    private void createPlayersLine(final TextField playerField) {
+        final Label players = new Label("Amount of players: ");
+        players.setFont(font);
+
+        playerField.setMaxWidth(maxWidth);
+
+        fillInPane.add(players, 0, 0);
+        fillInPane.add(playerField, 1, 0);
+    }
+
+    /**
+     * Adds the line to the pane to fill in the amount of chests.
+     * @param chestField the field in which the amount of chests need to be filled in
+     */
+    private void createChestsLine(final TextField chestField) {
+        final Label chests = new Label("Amount of chests: ");
+        chests.setFont(font);
+
+        chestField.setMaxWidth(maxWidth);
+
+        fillInPane.add(chests, 0, 1);
+        fillInPane.add(chestField, 1, 1);
+    }
+
```

Comment:
createPlayersLine and createChestsLine seem to be very similar methods, isn't there a ways to improve this

Reply:
Yes, maybe give a boolean or string to the method to differentiate between the two strings and place within the pane.